sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2472,30 +2466,31 @@
       }
     },
     "Twin": {
-      "description": "Twin Representation",
+      "description": "Twin Representation of the device/ module state information. Implicitly created and deleted when the corresponding device/ module identity is created or deleted in IoT Hub.",
```

Comment:
Nit: using the word "Twin" when describing the object "Twin" is a little confusing. A definition should not insist upon itself. Maybe "The service-side representation of the..." or just "The device/module state information"

Reply:
or "The state information for a device or module. Implicitly..."

Reply:
Sure